### PR TITLE
Saved DriverEntry arguments

### PR DESCRIPTION
**Pull request by belyshevdenis**
Added kmtest::GetDriverObject
Added kmtest::GetRegistryPath

### CODE REVIEW

File: include/kmtest/kmtest.h [hunk 1]
```diff
@@ -59,6 +59,12 @@
 #pragma section("KMTEST$__z", read)
 #pragma comment(linker, "/merge:KMTEST=.rdata")
 
+namespace
+{
+    DRIVER_OBJECT* s_driverObject = nullptr;
```
Comment (SergiusTheBest) [inline include/kmtest/kmtest.h:64]: Add `__declspec(selectany)` and put into kmtest namespace

File: include/kmtest/kmtest.h [hunk 2]
```diff
@@ -183,6 +189,16 @@ namespace kmtest
 
         return failures;
     }
+
+    DRIVER_OBJECT* GetDriverObject()
```
Comment (SergiusTheBest) [inline include/kmtest/kmtest.h:193]: use camelCase

File: include/kmtest/kmtest.h [hunk 3]
```diff
@@ -183,6 +189,16 @@ namespace kmtest
...
+    {
+        return s_driverObject;
+    }
+
+    PUNICODE_STRING GetRegistryPath()
```
Comment (SergiusTheBest) [inline include/kmtest/kmtest.h:198]: use camelCase

File: include/kmtest/kmtest.h (hunk 1 above)
Comment (SergiusTheBest) [inline include/kmtest/kmtest.h:64]: I think it's better to use g_ prefix

Comment (belyshevdenis) [reply include/kmtest/kmtest.h:64]: Removed the functions, because now the globals are available directly from the kmtest namespace

**Pull request merged**